File: bg/web/src/api/example/withdrawAudit.js
import service from '@/utils/request'

export const getWithdrawAuditList = (params) => {
  return service({
    url: '/withdrawAudit/getWithdrawAuditList',
    method: 'get',
    params
  })
}

export const approveWithdrawOrder = (data) => {
  return service({
    url: '/withdrawAudit/approve',
    method: 'post',
    data
  })
}

export const rejectWithdrawOrder = (data) => {
  return service({
    url: '/withdrawAudit/reject',
    method: 'post',
    data
  })
}

File: bg/server/router/example/withdraw_audit.go
package example

import (
	"github.com/flipped-aurora/gin-vue-admin/server/middleware"
	"github.com/gin-gonic/gin"
)

type WithdrawAuditRouter struct{}

func (r *WithdrawAuditRouter) InitWithdrawAuditRouter(Router *gin.RouterGroup, PublicRouter *gin.RouterGroup) {
	withdrawAuditRouter := Router.Group("withdrawAudit").Use(middleware.OperationRecord())
	withdrawAuditRouterWithoutRecord := Router.Group("withdrawAudit")
	_ = PublicRouter

	withdrawAuditRouterWithoutRecord.GET("getWithdrawAuditList", withdrawAuditApi.GetWithdrawAuditList)
	withdrawAuditRouter.POST("approve", withdrawAuditApi.ApproveWithdrawOrder)
	withdrawAuditRouter.POST("reject", withdrawAuditApi.RejectWithdrawOrder)
}
